cmd/end2endtest/helpers.go

```diff
@@ -306,11 +339,12 @@ func (t *e2eElection) setupElection(ed *vapi.ElectionDescription) error {
 		return err
 	}
 
-	t.proofs = t.generateProofs(root, ed.ElectionType.Anonymous)
+	t.proofs = t.generateProofs(root, anonymousVoting)
 
 	return nil
 }
 
+// overwriteVote allow to try to overwrite a previous vote given the index of the account, it can use the sameBlock or the nextBlock
```

Comment:
this func comment also looks unrelated. being just a comment, it would not merit a PR, but if you're doing a PR for the overwrite fixes (sameBlock vs nextBlock) please move these changes to that PR as well.